doc/stages/readers.ept.rst

```diff
@@ -58,4 +61,7 @@ threads
 .. _Entwine: https://entwine.io/
 .. _Potree: http://potree.entwine.io/data/nyc.html
 .. _Plasio: http://speck.ly/?s=http%3A%2F%2Fc%5B0-7%5D.greyhound.io&r=ept%3A%2F%2Fna.entwine.io%2Fnyc&ca=-0&ce=49.06&ct=-8239196%2C4958509.308%2C337&cd=42640.943&cmd=125978.13&ps=2&pa=0.1&ze=1&c0s=remote%3A%2F%2Fimagery%3Furl%3Dhttp%3A%2F%2Fserver.arcgisonline.com%2FArcGIS%2Frest%2Fservices%2FWorld_Imagery%2FMapServer%2Ftile%2F%7B%7Bz%7D%7D%2F%7B%7By%7D%7D%2F%7B%7Bx%7D%7D.jpg
+.. _Dimensions: https://pdal.io/dimensions.html
+.. _Schema: https://entwine.io/entwine-point-tile.html#schema
+.. _EPT writer: https://pdal.io/stages/writers.ept.html
```

Comment:
Now writers.ept_addon.html.

Reply:
Addressed in Addressed in a0f29158c966cafb1e64d886f213d3b4a2ae63e6.